neqo-crypto/build.rs

```diff
@@ -371,7 +369,7 @@ fn setup_for_gecko() -> Vec<String> {
     let flags_path = TOPOBJDIR.join("netwerk/socket/neqo/extra-bindgen-flags");
 
     println!("cargo:rerun-if-changed={}", flags_path.to_str().unwrap());
-    flags = fs::read_to_string(flags_path)
+    let mut flags: Vec<String> = fs::read_to_string(flags_path)
         .expect("Failed to read extra-bindgen-flags file")
         .split_whitespace()
         .map(std::borrow::ToOwned::to_owned)
```

Comment:
it be better to use this?

```suggestion
        .map(String::from)
```

And then (below):

```rust
   .collect::<Vec<_>>();
```

Reply:
Yup! That works too